torch/_export/serde/serialize.py

```diff
@@ -995,32 +1012,50 @@ def serialize_hoo_outputs(self, node: torch.fx.Node) -> List[Argument]:
             # a singleton tensor, we will serialize this to be a singleton
             # tensor list so that the deserializer knows to insert getitem nodes.
 
-            idx_to_name = {}
-            for user in node.users:
-                if user.target is not operator.getitem:
-                    continue
-                idx_to_name[user.args[1]] = user.name
-
-            for idx in range(len(meta_val)):
-                # FX does not emit a getitem node for any outputs that are unused.
-                # However, we need a name for them so that the number of outputs will
-                # correctly match the schema. Just assign a dummy name.
-                if idx not in idx_to_name:
-                    idx_to_name[idx] = f"{node.name}_unused_{idx}"
-
             if len(meta_val) == 1:
                 tensors = []
                 for i, v in enumerate(meta_val):
```

Comment:
If len(meta_val) == 1, we don't need to iterate on it?

Reply:
It could be a Tensor list